### PR TITLE
PHP 7 Travis

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Doesn't look like the CI is actually enabled.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
   - 5.5
   - 5.6
-  - hhvm
-  - hhvm-nightly
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+  - 7.4
+  - hhvm-3.18
```
Comment (barryvdh) [inline .travis.yml:14]: I think we can just drop HHVM support

**Comment by barryvdh** [comment]
I've enabled Travis, but we need to update the environment for 5.3:
PHP 5.3.3 is supported only on Precise.

**Comment by barryvdh** [comment]
Or just drop it

**Comment by barryvdh** [comment]
Just to be clear, I meant drop 5.3 support, not the PR. I appreciate the PR :)

**Comment by GrahamCampbell** [comment]
Yeh, I know. Was gonna close and re-open to trigger travis. Turns out I can't re-open it again, so need to re-create it.

**Pull request closed without merging**